src/hooks/motionWidgets/stakingWidget/useStakingControls.ts

```diff
@@ -0,0 +1,43 @@
+import { useFormContext } from 'react-hook-form';
+
+import {
+  getStakeFromSlider,
+  SLIDER_AMOUNT,
+  useStakingWidgetContext,
+} from '~common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget';
+
+const useStakingControls = (limitExceeded: boolean) => {
+  const {
+    enoughReputation,
+    nayPercentage,
+    canBeStaked,
+    remainingToStake,
+    minUserStake,
+    getErrorType,
+    setIsSummary,
+  } = useStakingWidgetContext();
+  const { getValues } = useFormContext();
+  const sliderAmount: number = getValues(SLIDER_AMOUNT);
+  const stake = getStakeFromSlider(
+    sliderAmount,
+    remainingToStake,
+    minUserStake,
+  );
+
+  const showObjectButton = nayPercentage === '0.00';
```

Comment:
Once you object to a motion, the flow changes so that all future stakes are mediated via the GroupedTotal stake component, thus rendering the object button unnecessary.